concent_api/core/decorators.py

```diff
@@ -122,8 +124,9 @@ def wrapper(request: HttpRequest, *args: Any, **kwargs: Any) -> Union[HttpRespon
                 log(logger, 'Error:', exception.__class__.__name__)
                 return JsonResponse({'error': f'{exception}'}, status = 400)
             except MessageError as exception:
-                log(logger, ERROR_IN_GOLEM_MESSAGE, exception.__class__.__name__)
-                return JsonResponse({'error': join_messages(ERROR_IN_GOLEM_MESSAGE, str(exception))}, status = 400)
+                request_id = generate_uuid()
+                log(logger, f'uuid: {request_id}', ERROR_IN_GOLEM_MESSAGE, exception.__class__.__name__, str(traceback.format_exc()))
+                return JsonResponse({'error': join_messages(f'uuid: {request_id}', ERROR_IN_GOLEM_MESSAGE, exception.__class__.__name__)}, status=400)
```

Comment:
why `exception.__class__.__name__` instead of `str(exception)`?

Reply:
`str(exception)` is empty string, name of exception is  `exception.__class__.__name__ `. It is really don't necessary now because of stacktrace, but I would leave it as more readable